src/main/java/org/synyx/urlaubsverwaltung/application/statistics/ApplicationForLeaveStatisticsBuilder.java

```diff
@@ -58,56 +55,35 @@ public ApplicationForLeaveStatistics build(Person person, LocalDate from, LocalD
         Assert.isTrue(from.getYear() == to.getYear(), "From and to must be in the same year");
 
         final ApplicationForLeaveStatistics statistics = new ApplicationForLeaveStatistics(person, vacationTypeService);
-        final Optional<Account> account = accountService.getHolidaysAccount(from.getYear(), person);
-        if (account.isPresent()) {
-            final BigDecimal vacationDaysLeft = vacationDaysService.calculateTotalLeftVacationDays(account.get());
-            statistics.setLeftVacationDays(vacationDaysLeft);
-        }
+        statistics.setLeftVacationDays(calculateLeftVacationDays(person, from));
+        statistics.setLeftOvertime(overtimeService.getLeftOvertimeForPerson(person));
 
         final List<Application> applications = applicationService.getApplicationsForACertainPeriodAndPerson(from, to, person);
         for (Application application : applications) {
             if (application.hasStatus(WAITING) || application.hasStatus(TEMPORARY_ALLOWED)) {
-                statistics.addWaitingVacationDays(application.getVacationType(), getVacationDays(application, from.getYear()));
+                statistics.addWaitingVacationDays(application.getVacationType(), getVacationDaysFor(application, from, to));
             } else if (application.hasStatus(ALLOWED) || application.hasStatus(ALLOWED_CANCELLATION_REQUESTED)) {
-                statistics.addAllowedVacationDays(application.getVacationType(), getVacationDays(application, from.getYear()));
+                statistics.addAllowedVacationDays(application.getVacationType(), getVacationDaysFor(application, from, to));
             }
         }
 
-        statistics.setLeftOvertime(overtimeService.getLeftOvertimeForPerson(person));
         return statistics;
     }
 
-    private BigDecimal getVacationDays(Application application, int relevantYear) {
-
-        final int yearOfStartDate = application.getStartDate().getYear();
-        final int yearOfEndDate = application.getEndDate().getYear();
+    private BigDecimal getVacationDaysFor(Application application, LocalDate from, LocalDate to) {
 
         final DayLength dayLength = application.getDayLength();
         final Person person = application.getPerson();
 
-        if (yearOfStartDate != yearOfEndDate) {
```

Comment:
Hier wurden die Arbeitstage im relevanten Jahr berechnet. Wie ist das bei einem Antrag über die Jahresgrenze hinaus?

Reply:
Getestet sieht in der Urlaubstatistik gut aus